### DISCUSSION TITLE
Has anyone ever used riscv bare_metal workload?

### DISCUSSION THREAD

**Discussion by flyeagle0** [gem5/gem5#897] [Q&A] [answered]
How can I use this workload in the path of src/arch/riscv/bare_metal/fs_workload.cc? Has anyone ever used it?

**Comment by BobbyRBruce** [comment, marked as answer]
There appears to be an example script which uses this: configs/example/riscv/fs_linux.py.

This configuration script has the option `--bare-metal` which, from what I can see, enables bare-metal workloads using src/arch/riscv/bare_metal/fs_workload.cc.

I haven't tried running this script. By looking at the git log, it has been updated to work with the latest version but whether bare_metal was tested is unknown.
